R/return_linter.R

```diff
@@ -0,0 +1,206 @@
+#' Return linter
+#'
+#' This linter checks for explicit [return()] at the end of a function
+#'
+#' @param use_implicit_returns Whether to use implicit or explicit returns
+#'
+#' @param additional_allowed_func Names of additional functions that are
```

Comment:
Better name `return_functions`?

Reply:
Maybe "additional_return_functions"?

Reply:
It seems overly verbose to me. Maybe a good way would be to document the base return functions while keeping the argument name short.

Reply:
I like `return_calls` or `return_functions`.